rogue-thi-app/components/cards/FoodCard.js

```diff
@@ -53,18 +53,17 @@ export default function FoodCard () {
           .find(x => x.timestamp === today)
           ?.meals
           .filter(x => x.category !== 'Suppe' && selectedRestaurants.includes(x.restaurant.toLowerCase()))
+        const sortedEntries = todayEntries ? todayEntries.sort((a, b) => userMealRating(b) - userMealRating(a)) : []
```

Comment:
This could be improved by just calling `todayEntries?.sort` (optional chaining operator) 😊 then you would not need to introduce a new variable

Reply:
Oh, you're right xD. Fixed it.